### ISSUE TITLE
ViewStack: this.transition not used as in the documentation

### ISSUE THREAD

**Issue by EAlexRojas** [ibm-js/deliteful#529]
According to the documentation, when calling show, showNext or showPrevious without the params parameter, this.transtition and this.reverse should be used. But before doing the transition, in the changeDisplay method line [252](https://github.com/ibm-js/deliteful/blob/master/ViewStack.js#L252) :
```javascript
 var transition  = (origin === widget) ? "none" : (event.transition || "slide");
```
If transition is not specified, it's setted to "slide" instead of using this.transition value.

With this, if, for example, one creates a viewStack as follows:
```javascript
<d-view-stack id="vs" transition="flip">
...
</d-view-stack>
```
And show, showNext or showPrevious are used without params, this "flip" transition is ignored and the "slide" transition is used instead.

**Comment by cjolif** [comment]
Maybe you can fix that as well part of #530 and add the corresponding missing tests?

**Comment by EAlexRojas** [comment]
Ok
